repo/packages/C/chronos/5/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "packagingVersion": "4.0",
+  "name": "chronos",
+  "version": "2.5.0",
+  "scm": "https://github.com/mesos/chronos.git",
+  "maintainer": "support@mesosphere.io",
+  "description": "A fault tolerant job scheduler for Mesos which handles dependencies and ISO8601 based schedules.",
+  "framework": true,
+  "minDcosReleaseVersion": "1.9",
```

Comment:
@gkleiman you only want this to be available in `minDcosReleaseVersion ` >= 1.9 ?